cargo-generate.toml

```diff
@@ -3,3 +3,18 @@
 # This is needed when files contains curly brackets as in `v4-cargo-cache-{{ arch }}-{{ checksum "Cargo.lock" }}`.
 # The files will be copied 1:1 to the target project. To avoid shipping them completely add them to `.genignore`.
 exclude = [".circleci/config.yml"]
+
+[placeholders.minimal]
+type = "bool"
+prompt = "Would you like to generate the minimal template?"
```

Comment:
Is it possible to provide a little more info here? Something along these lines:

```
The full template includes some example logic in case you're new to CosmWasm smart contracts.
The minimal template assumes you already know how to write your own logic, and doesn't get in your way.
```

Just don't want anyone to feel lost.

Reply:
Sure, sounds good.